nextcloud/viewer
# fix: use filename to find current index

Fix #484 

The viewer component is usually used with a folder and the basename unique. 
However in photos, the fileList contains objects from multiple folders and the basename is not always unique.

How to reproduce:

- Create folder A and B
- Upload an image to folder A and name it "Test"
- Upload another image to folder B and also name it "Test"
- Open photos
- Click on the image from folder B
- The image from folder A is shown

To find the correct "currentIndex" we need filename (including the path).

My current understand is that fileInfo.filename should be always there, even if the remote file is not a dav file (e.g. attachments from mail). If that's not the case, we need to come up with an alternative solution.
